machine_spec.py

```diff
@@ -284,7 +286,8 @@ def detect_arch() -> str:
     "armbe8",
     "mips",
     "mips64",
+    "powerpc",
     "s390x",
 }
 
-TARGET_TRIPLET_ARCH_PATTERN = re.compile(r"^(i.86|x86_64|arm(v\w+)?|aarch64|mips\w*|s390x)$")
+TARGET_TRIPLET_ARCH_PATTERN = re.compile(r"^(i.86|x86_64|arm(v\w+)?|aarch64|mips\w*|powerpc|s390x)$")
```

Comment:
This is correct though -- assuming this is what the distro-supplied toolchains use in their triplet.